authentication-tokens/sdk_examples.cs

```diff
@@ -0,0 +1,15 @@
+// .NET SDK examples for generating Aurora DSQL authentication tokens
+
+// --8<-- [start:dotnet-generate-token]
+using Amazon;
+using Amazon.DSQL.Util;
+using Amazon.Runtime;
+
+var yourClusterEndpoint = "insert-dsql-cluster-endpoint";
+
+AWSCredentials credentials = FallbackCredentialsFactory.GetCredentials();
+
+var token = DSQLAuthTokenGenerator.GenerateDbConnectAdminAuthToken(credentials, RegionEndpoint.USEast1, yourClusterEndpoint);
```

Comment:
Several of the other samples have a line like:

```
// Use `generateDbConnectAuthToken` if you are _not_ logging in as `admin` user
```

Is that function not provided in the .NET SDK, or should we add it here? Same comment applies to the C++ sample.

Reply:
I think this can fall under a separate correctness/polishing pass as discussed [here](https://github.com/aws-samples/aurora-dsql-samples/pull/216#discussion_r2632414243). Edit: oh yeah I see your comment above with the same ^